corpus/templates/base.html

```diff
@@ -45,20 +45,68 @@
 
 <div class="navbar bg-primary text-primary-content sticky top-0 z-10">
```

Comment:
The navbar is not responsive. When the viewport width reduces below the navbar width, it should collapse into a hamburger menu similar to the existing navbar.
To test this out locally, try reducing your browser width to minimum and see how the navbar looks with smaller browser width. Additionally, for any UI changes I would strongly recommend using the Device Emulator tool that comes with Chromium based browsers ([see here](https://developer.chrome.com/docs/devtools/device-mode)). This helps ensure that our webpages look good on all devices.